### PR TITLE
Fix panic after calling DELETE /contracts/all

### PR DESCRIPTION
**Pull request by mike76-dev**

### CODE REVIEW

File: internal/stores/metadata.go [hunk 1]
```diff
@@ -458,7 +458,9 @@ func (s *SQLStore) RemoveContracts(ctx context.Context) error {
 	if err := s.db.Where("TRUE").Delete(&dbContract{}).Error; err != nil {
 		return err
 	}
-	s.knownContracts = nil
+	for c := range s.knownContracts {
+		delete(s.knownContracts, c)
+	}
```
Comment (ChrisSchinnerl) [inline internal/stores/metadata.go:463]:
```suggestion
	s.knownContracts = make(map[types.FileContractID]struct{})
```

**Comment by mike76-dev** [comment]
Do the tests restart after some time, or does someone have to push them manually?

**Comment by ChrisSchinnerl** [comment]
You need to push a new commit or do `git commit --amend` and force push your latest commit again @mike76-dev

**Comment by ChrisSchinnerl** [comment]
Cherry-picked your fix to https://github.com/SiaFoundation/renterd/pull/235

**Pull request closed without merging**